### PR TITLE
add warning when remove all data

### PR DESCRIPTION
**Pull request by joanasouza**
Issue #119

### CODE REVIEW

File: js/lightbeam.js [hunk 1]
```diff
@@ -142,7 +142,10 @@ const lightbeam = {
   resetData() {
     const resetData = document.getElementById('reset-data-button');
     resetData.addEventListener('click', async () => {
-      await storeChild.reset();
+      var confirmation = confirm('Pressing OK will delete all'
+       + ' Lightbeam information. Are you sure?');
+      if (confirmation == true)
+        await storeChild.reset();
       window.location.reload();
     });
```
Comment (princiya) [inline js/lightbeam.js]: @joanasouza thanks for the PR. please enclose the `if` 
```
if (confirmation === true) {
 -- await
 --reload
}
```

Comment (princiya) [reply js/lightbeam.js]: `confirm` gives true/false values. please make if statement as `if (confirmation)`

**Comment by princiya** [comment]
I shall create a follow-up issue to add CSS to this confirm dialog box.

File: js/lightbeam.js (hunk 1 above)
Comment (princiya) [reply js/lightbeam.js]: also, please change `confirmation` to `const`. Thanks :)

Comment (joanasouza) [reply js/lightbeam.js]: Thanks, already applied the changes 😄

**Comment by jonathanKingston** [comment]
Sorry to be pedantic but can you make a new variable  for the confirm
string rather than using string concatenation to get under the line limit.
If that is still too long you could use a template string also.

Thanks for the patch! Let me know if you need any help with anything else.

File: js/lightbeam.js [hunk 2]
```diff
@@ -142,8 +142,12 @@ const lightbeam = {
...
     const resetData = document.getElementById('reset-data-button');
     resetData.addEventListener('click', async () => {
-      await storeChild.reset();
-      window.location.reload();
+      const confirmation = confirm('Pressing OK will delete all'
+       + ' Lightbeam information. Are you sure?');
+      if (confirmation) {
+        await storeChild.reset();
+        window.location.reload();
+      }
```
Comment (princiya) [inline js/lightbeam.js]: @joanasouza as per @jonathanKingston 's we could do it like -
`messagePart1` & `messagePart2`?
confirm(`${m1} ${m2}`);

**Comment by jonathanKingston** [comment]
I'm going to merge this however some follow up work could be:
- Changing to meaningful variable names (rather than start and end)
- Do the concat in the template string (rather than having spaces in the strings):
`${msgDeletion} ${msgConfirm}`

**Pull request merged**